packages/carbon-component-mapper/src/tests/time-picker.test.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { mount } from 'enzyme';
 
-import FormRenderer, { componentTypes } from '@data-driven-forms/react-form-renderer';
+import { FormRenderer, componentTypes } from '@data-driven-forms/react-form-renderer';
```

Comment:
Oh, this will probably also cause issues in tests.

Reply:
Is the original line correct?

Reply:
Yeah, in v2 FormRenderer is exported as the default